### PR TITLE
Fix panic when no connection during reload

### PR DESCRIPTION
**Pull request by atluq**
**Checklist**

- [x] commits conform to the [Commit message format](https://github.com/qlik-oss/gopherciser/blob/master/.github/CONTRIBUTING.md#commit)

**Squash commit message**

Error instead of panic if no connection during reload

**Info**

Closes #675

### CODE REVIEW

**Review by DnlLrssn — APPROVED** [review]

**Comment by DnlLrssn** [comment]
hmm wait, maybe we should check connection too?

**Comment by atluq** [comment]
Yes, let's do that.

File: scenario/reload.go [hunk 1]
```diff
@@ -101,8 +106,12 @@ func (settings ReloadSettings) Execute(sessionState *session.State,
 				// Get the progress using the request id we reserved for the reload
 				var progress *enigma.ProgressData
 				getProgress := func(ctx context.Context) error {
+					uplink := sessionState.Connection.Sense()
```
Comment (DnlLrssn) [inline scenario/reload.go:109]: still should check sessionState.Connection != here as this is in separate go-routine and something else could nil the connection.

Comment (atluq) [reply scenario/reload.go:109]: ahh, then getting an uplink from sessionState should be atomic or we should just accept the panics

**Review by DnlLrssn — APPROVED** [review]

**Pull request merged**